lib/fuser_basic.h

```diff
@@ -49,7 +51,7 @@ struct BasicGateFuser final {
    * applied together. Note that gates fused with this method are not
    * multiplied together until ApplyFusedGate is called on the output.
    * @param num_qubits The number of qubits acted on by 'gates'.
-   * @param gates The gates to be fused.
+   * @param gates The gates to be fused. Gate times should be ordered.
```

Comment:
This is a change from our previous assumptions, correct? re: #74

Reply:
Correct. Gates don't get explicitly sorted. So there was a possibility of getting incorrect results for unordered gate times.